docs/pages/versions/unversioned/sdk/shared-element.mdx

```diff
@@ -8,7 +8,7 @@ import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
 import Video from '~/components/plugins/Video';
 
-> **info** This library is listed in the Expo SDK reference because it is included in [Expo Go](/get-started/expo-go/). You may use any library of your choice with [development builds](/develop/development-builds/introduction/).
+> **info** This library is no longer available in [Expo Go](/get-started/expo-go/). You can still use it with [development builds](/develop/development-builds/introduction/).
```

Comment:
cc @brentvatne if you want to remove the doc entirely in the future.

Reply:
yeah that makes sense. let's remove this doc from unversioned and go back to sdk 48 and add this note to the top of the page

> **info** In SDK 49 and higher, this library will no longer available in [Expo Go](/get-started/expo-go/). You can still use it with [development builds](/workflow/customizing/).